Meta/run.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
```

Comment:
Before we merge this PR, we probably want to remove the logic in this script? I think it's a bit pointless to maintain both scripts?

Reply:
See comment from me above